### PR TITLE
[Docathon][Add API Legend No.2]

### PR DESCRIPTION
**Pull request by Liyulingyue**
### PR Category

User Experience

### PR Types

Docs

### Description

update doc for https://github.com/PaddlePaddle/docs/pull/6941

### CODE REVIEW

File: python/paddle/tensor/manipulation.py [hunk 1]
```diff
@@ -1538,6 +1538,20 @@ def broadcast_tensors(
...
 
+    The following figure illustrates the process of broadcasting three tensors to the same dimensions. 
+    The dimensions of the three tensors are [4, 1, 3], [2, 3], and [4, 2, 1], respectively. During broadcasting, 
+    alignment starts from the last dimension, and for each dimension, either the sizes of the two tensors in that dimension are equal, 
+    or one of the tensors has a dimension of 1, or one of the tensors lacks that dimension. In the figure below, in the last dimension, 
+    Tensor3 has a size of 1, while Tensor1 and Tensor2 have sizes of 3; thus, this dimension is expanded to 3 for all tensors. 
+    In the second-to-last dimension, Tensor1 has a size of 2, and Tensor2 and Tensor3 both have sizes of 2; hence, this dimension is expanded to 2 for all tensors. 
+    In the third-to-last dimension, Tensor2 lacks this dimension, while Tensor1 and Tensor3 have sizes of 4; consequently, 
+    this dimension is expanded to 4 for all tensors. Ultimately, all tensors are expanded to [4, 2, 3].
+
```
Comment (Liyulingyue) [inline python/paddle/tensor/manipulation.py:1549]:
```suggestion


```

File: python/paddle/tensor/manipulation.py [hunk 2]
```diff
@@ -1538,6 +1538,21 @@ def broadcast_tensors(
...
+
```
Comment (Liyulingyue) [inline python/paddle/tensor/manipulation.py]:
```suggestion

```

**Comment by luotao1** [comment]
张师傅，请过一下 CodeStyle流水线

File: python/paddle/tensor/manipulation.py [hunk 3]
```diff
@@ -1538,6 +1538,19 @@ def broadcast_tensors(
...
+    or one of the tensors has a dimension of 1, or one of the tensors lacks that dimension. In the figure below, in the last dimension,
+    Tensor3 has a size of 1, while Tensor1 and Tensor2 have sizes of 3; thus, this dimension is expanded to 3 for all tensors.
+    In the second-to-last dimension, Tensor1 has a size of 2, and Tensor2 and Tensor3 both have sizes of 2; hence, this dimension is expanded to 2 for all tensors.
+    In the third-to-last dimension, Tensor2 lacks this dimension, while Tensor1 and Tensor3 have sizes of 4; consequently,
+    this dimension is expanded to 4 for all tensors. Ultimately, all tensors are expanded to [4, 2, 3].
+    .. image:: https://githubraw.cdn.bcebos.com/PaddlePaddle/docs/develop/docs/images/api_legend/broadcast.png
+       :width: 800
+       :alt: Illustration of BroadCast
+       :align: center
+
```
Comment (Liyulingyue) [inline python/paddle/tensor/manipulation.py:1554]:
```suggestion


```

File: python/paddle/tensor/manipulation.py [hunk 4]
```diff
@@ -1538,6 +1538,20 @@ def broadcast_tensors(
...
+
```
Comment (Liyulingyue) [inline python/paddle/tensor/manipulation.py]:
```suggestion

```

File: python/paddle/tensor/manipulation.py [hunk 5]
```diff
@@ -1538,6 +1538,18 @@ def broadcast_tensors(
...
+    alignment starts from the last dimension, and for each dimension, either the sizes of the two tensors in that dimension are equal,
+    or one of the tensors has a dimension of 1, or one of the tensors lacks that dimension. In the figure below, in the last dimension,
+    Tensor3 has a size of 1, while Tensor1 and Tensor2 have sizes of 3; thus, this dimension is expanded to 3 for all tensors.
+    In the second-to-last dimension, Tensor1 has a size of 2, and Tensor2 and Tensor3 both have sizes of 2; hence, this dimension is expanded to 2 for all tensors.
+    In the third-to-last dimension, Tensor2 lacks this dimension, while Tensor1 and Tensor3 have sizes of 4; consequently,
+    this dimension is expanded to 4 for all tensors. Ultimately, all tensors are expanded to [4, 2, 3].
+    .. image:: https://githubraw.cdn.bcebos.com/PaddlePaddle/docs/develop/docs/images/api_legend/broadcast.png
+       :width: 800
+       :alt: Illustration of BroadCast
+       :align: center
```
Comment (Liyulingyue) [inline python/paddle/tensor/manipulation.py:1553]:
```suggestion
       :align: center

```

File: python/paddle/tensor/manipulation.py [hunk 6]
```diff
@@ -1538,6 +1538,19 @@ def broadcast_tensors(
...
 
+    The following figure illustrates the process of broadcasting three tensors to the same dimensions.
+    The dimensions of the three tensors are [4, 1, 3], [2, 3], and [4, 2, 1], respectively. During broadcasting,
+    alignment starts from the last dimension, and for each dimension, either the sizes of the two tensors in that dimension are equal,
+    or one of the tensors has a dimension of 1, or one of the tensors lacks that dimension. In the figure below, in the last dimension,
+    Tensor3 has a size of 1, while Tensor1 and Tensor2 have sizes of 3; thus, this dimension is expanded to 3 for all tensors.
+    In the second-to-last dimension, Tensor1 has a size of 2, and Tensor2 and Tensor3 both have sizes of 2; hence, this dimension is expanded to 2 for all tensors.
+    In the third-to-last dimension, Tensor2 lacks this dimension, while Tensor1 and Tensor3 have sizes of 4; consequently,
+    this dimension is expanded to 4 for all tensors. Ultimately, all tensors are expanded to [4, 2, 3].
+    .. image:: https://githubraw.cdn.bcebos.com/PaddlePaddle/docs/develop/docs/images/api_legend/broadcast.png
```
Comment (sunzhongkai588) [inline python/paddle/tensor/manipulation.py]:
```suggestion
    this dimension is expanded to 4 for all tensors. Ultimately, all tensors are expanded to [4, 2, 3].

    .. image:: https://githubraw.cdn.bcebos.com/PaddlePaddle/docs/develop/docs/images/api_legend/broadcast.png
```

**Review by luotao1 — APPROVED** [review]

**Pull request merged**